src/platform/converters.ts

```diff
@@ -29,3 +41,66 @@ export function currencyToPlatformCurrency(
     })),
   };
 }
+
+export const getPlatformTransactionSignFlowInfos = (
```

Comment:
Isn't this the ideal place for something that gets exported `byFamily` and we just provide the family to the object to get the method? Like we do with any of the generated code by family, ie https://github.com/LedgerHQ/ledger-live-common/blob/master/src/generated/hw-getAddress.ts you would then import this byFamily and pas the platform 🤷🏼‍♂️

Reply:
Maybe, I'm not yet familiar with this `byFamily` kind of abstraction. I'll take a look at this and propose an update accordingly

Reply:
Updated here e0c3b5ffeaea9e200421fab8bc1afb075b971dcf